lib/plugins/create/templates/hello-world/serverless.yml

```diff
@@ -21,4 +21,3 @@ functions:
       - http:
           path: hello-world
           method: get
-          cors: true
```

Comment:
Thanks for the comments. But it's a little hard to follow the other issues / PRs.

Let me see if I get your motivation for CORS correctly: "If you expose a function via an http endpoint this will likely be consumed by some frontend javascript. Which will likely be served from another endpoint and therefor result in CORS problems."

If that is the case than we should show that in the hello-world example and serve some frontend javascript and with this motivate CORS. 

I would be in the camp of keeping the http enabled hello-world as simple as possible and in a second step (e.g. e.g. hello-html-world) add CORS and motivate it.

(Sorry for making this debate so big about a small change. I'm just stating my thoughts. Feel free to close / pause this PR.)